### PR TITLE
Update bundle with latest version CoreBundle

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

**Review by rjzondervan — APPROVED** [review]

File: Installation/Action/hp.HandleSendEmailAction.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://hp.nl/action/hp.HandleSendEmailAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.4",
```
Comment (WilcoLouwerse) [inline Installation/Action/hp.HandleSendEmailAction.json:4]: Could be 0.0.1

File: Installation/Action/hp.InvitePartnerAction.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "$id": "https://hp.nl/action/hp.InvitePartnerAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.4",
```
Comment (WilcoLouwerse) [inline Installation/Action/hp.InvitePartnerAction.json:4]: Could be 0.0.1

File: Installation/Action/hp.InviteWitnessAction.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+  "$id": "https://hp.nl/action/hp.InviteWitnessAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.4",
```
Comment (WilcoLouwerse) [inline Installation/Action/hp.InviteWitnessAction.json:4]: Could be 0.0.1

File: Installation/SecurityGroup/assent.securityGroup.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
   "$id": "https://huwelijksplanner.nl/SecurityGroup/assent.securityGroup.json",
   "$schema": "https://docs.commongateway.nl/schemas/SecurityGroup.schema.json",
-  "version": "0.0.",
+  "version": "0.0.2",
```
Comment (WilcoLouwerse) [inline Installation/SecurityGroup/assent.securityGroup.json:4]: Could be 0.0.1 (probably 🤔)

File: src/Service/InvitePartnerService.php [hunk 5]
```diff
@@ -131,6 +131,71 @@ private function createEmailAndSmsData(ObjectEntity $requester, ObjectEntity $pe
...
+        return $partnerDetails;
+
+    }//end invitePartnerLogin()
+
+
+    /**
+     * This function gets the partner details from the given bsn.
+     *
+     * @param ObjectEntity $huwelijkObject The huwelijk from the request.
+     * @param string       $bsn            The bsn of the given partner via email.
```
Comment (WilcoLouwerse) [inline src/Service/InvitePartnerService.php:173]: You no longer have this param in your function

File: src/Service/InvitePartnerService.php [hunk 6]
```diff
@@ -131,6 +131,71 @@ private function createEmailAndSmsData(ObjectEntity $requester, ObjectEntity $pe
...
+     */
+    private function invitePartnerInvite(ObjectEntity $huwelijkObject): array
+    {
+        $partnerDetails = [];
+
+        foreach ($huwelijkObject->getValue('partners') as $huwelijkPartner) {
+            if ($huwelijkPartner->getValue('contact')->getValue('subjectIdentificatie') === false) {
+                $partnerDetails['personAssent'] = $huwelijkPartner;
+                $partnerDetails['person']       = $huwelijkPartner->getValue('contact');
+            }//end if
```
Comment (WilcoLouwerse) [inline src/Service/InvitePartnerService.php:185]: (//end if or //end foreach is only necessary if the if statement is long, more than 5 lines long or something like that)

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: Installation/Action/hp.CreateMarriageAction.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://hp.nl/action/hp.CreateMarriageAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.4",
```
[the remark this answers is not in the document]
Comment (smisidjan) [reply Installation/Action/hp.CreateMarriageAction.json:4]: This is a action that was already in the repository. I only changed the location

File: Installation/Action/hp.HandleSendEmailAction.json (hunk 1 above)
Comment (smisidjan) [reply Installation/Action/hp.HandleSendEmailAction.json:4]: This is a action that was already in the repository. I only changed the location

File: Installation/Action/hp.InvitePartnerAction.json (hunk 2 above)
Comment (smisidjan) [reply Installation/Action/hp.InvitePartnerAction.json:4]: This is a action that was already in the repository. I only changed the location

File: Installation/Action/hp.InviteWitnessAction.json (hunk 3 above)
Comment (smisidjan) [reply Installation/Action/hp.InviteWitnessAction.json:4]: This is a action that was already in the repository. I only changed the location

File: Installation/Action/hp.CreateMarriageAction.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (rjzondervan) [reply Installation/Action/hp.CreateMarriageAction.json:4]: If the action is moved from installation.json it indeed should have a version number higher than the version number in installation.json

File: Installation/Action/hp.InvitePartnerAction.json (hunk 2 above)
Comment (rjzondervan) [reply Installation/Action/hp.InvitePartnerAction.json:4]: If the action is moved from installation.json it indeed should have a higher (or if the contents remain the same, the same) version number as in installation.json

File: Installation/Action/hp.InviteWitnessAction.json (hunk 3 above)
Comment (rjzondervan) [reply Installation/Action/hp.InviteWitnessAction.json:4]: If the action is moved from installation.json it indeed should have a higher (or if the contents remain the same, the same) version number as in installation.json

**Review by rjzondervan — APPROVED** [review]

**Review by bbrands02 — APPROVED** [review]

**Pull request merged**